### PR TITLE
Updates to navigation and URLs within the main header

### PR DESCRIPTION
**Pull request by AndrewCowle**
https://regulatorydelivery.atlassian.net/browse/COSBETA-1588

## Description
Updates to the main header. Added in a products-redirect on the responsible persons controller to use the concern to get a current responsible person. This ensures the nav works correctly when visiting a shared controller (My Account) without exposing the responsible person to search.

## Review apps

https://cosmetics-pr-xxxx-submit-web.london.cloudapps.digital/
https://cosmetics-pr-xxxx-search-web.london.cloudapps.digital/

## Checklist:

- [ ] Automated checks are passing locally.
- [ ] CHANGELOG updated if change is worth telling users about. Changes in notification wizard should always be included.
### General testing
- [ ] Test without javascript
- [ ] Test on small screen
### Accessibility testing
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable css - does content make sense and appear in a logical order?
- [ ] Passes automated checker (automated in build or manual)

### CODE REVIEW

File: cosmetics-web/.gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.env.development
```
Comment (slawosz) [inline cosmetics-web/.gitignore:1]: Some of us still use `.env.develepoment`

File: cosmetics-web/app/controllers/responsible_persons_controller.rb [hunk 2]
```diff
@@ -1,9 +1,13 @@
 class ResponsiblePersonsController < SubmitApplicationController
+  include ResponsiblePersonConcern
```
Comment (slawosz) [inline cosmetics-web/app/controllers/responsible_persons_controller.rb:2]: `ResponsiblePersonConcern` is already in `SubmitApplicationController`

File: cosmetics-web/app/controllers/responsible_persons_controller.rb [hunk 3]
```diff
@@ -36,6 +40,10 @@ def update
     end
   end
 
+  def products_redirect
```
Comment (slawosz) [inline cosmetics-web/app/controllers/responsible_persons_controller.rb:43]: I think better name would be `products_page_redirect`

File: cosmetics-web/.gitignore (hunk 1 above)
Comment (AndrewCowle) [reply cosmetics-web/.gitignore:1]: But this shouldn't be committed to git right - env.development.example should be there as a template for others to use, you could commit things like your notify key etc

Comment (slawosz) [reply cosmetics-web/.gitignore:1]: Yes, so now, without `.env.development` in `.gitignore` my `.env.development` is on untracked files list. We don't want to remove it as we want to ignore it.

**Review by slawosz — APPROVED** [review]

**Pull request merged**